creator/lib/screens/home_screen.dart

```diff
@@ -9,6 +11,38 @@ class HomeScreen extends StatelessWidget {
       appBar: AppBar(
         backgroundColor: Colors.green,
         title: Text('ホーム'),
+        iconTheme: IconThemeData(color: Colors.black),
+      ),
+      drawer: Drawer(
```

Comment:
this function is too long now. please break it down

Reply:
I break it down to separate method.